src/client.rs

```diff
@@ -929,7 +969,30 @@ impl<C: Config> Client<C> {
             C2sPlayPacket::SetHeldItemS2c(_) => {}
             C2sPlayPacket::ProgramCommandBlock(_) => {}
             C2sPlayPacket::ProgramCommandBlockMinecart(_) => {}
-            C2sPlayPacket::SetCreativeModeSlot(_) => {}
+            C2sPlayPacket::SetCreativeModeSlot(e) => {
+                if e.slot == -1 {
+                    // The client is trying to drop a stack of items
+                    match e.clicked_item {
+                        crate::protocol::Slot::Empty => log::warn!(
+                            "Invalid packet, creative client tried to drop a stack of nothing."
+                        ),
```

Comment:
Not sure if this is a good idea. Cheaters can spam the console without anticheat devs having any way to react to it. Might be better to externalize this possibility. If a log must be done, maybe it should be `log::trace!`?